htdocs/public/paypal/paymentok.php

```diff
@@ -109,6 +109,33 @@
 
 llxHeaderPaypal($langs->trans("PaymentForm"));
 
+// Show logo (search order: logo defined by PAYBOX_LOGO_suffix, then PAYBOX_LOGO, then small company logo, large company logo, theme logo, common logo)
+$width=0;
+// Define logo and logosmall
+$logosmall=$mysoc->logo_small;
+$logo=$mysoc->logo;
+$paramlogo='PAYBOX_LOGO_'.$suffix;
```

Comment:
Code is coming from paybox page, so constant "PAYBOX_" should be replaced with "PAYPAL_" here and everywhere a PAYBOX is found in thi page